geomesa-fs/geomesa-fs-storage/geomesa-fs-storage-orc/src/main/scala/org/locationtech/geomesa/fs/storage/orc/utils/OrcAttributeReader.scala

```diff
@@ -83,17 +82,21 @@ object OrcAttributeReader {
   }
 
   // noinspection LanguageFeature
-  private def createGeometryReader(binding: ObjectType, x: ColumnVector, y: ColumnVector, i: Int): OrcAttributeReader = {
+  private def createGeometryReader(binding: ObjectType, cols: Array[ColumnVector], col: Int, i: Int): OrcAttributeReader = {
     implicit def toDoubleColumnVector(vec: ColumnVector): DoubleColumnVector = vec.asInstanceOf[DoubleColumnVector]
     implicit def toListColumnVector(vec: ColumnVector): ListColumnVector = vec.asInstanceOf[ListColumnVector]
 
+    def x: ColumnVector = cols(col)
+    def y: ColumnVector = cols(col + 1)
```

Comment:
Does 'y' ever get called in the 'Geometry' call?  If so, then if the WKB geometry is the last column, could this throw an ArrayOutOfBounds exception?

Reply:
no, it doesn't get called in that case: https://github.com/locationtech/geomesa/pull/2564/files/43a17a89165231bd8c0ce23dff3acbb95ab60123#diff-3f8bf3d5ae3a2c8ea7ae82c2b49d6672R99